.github/workflows/hygiene-check.yml

```
@@ -0,0 +1,173 @@
+          python-version: '3.11'
+          
+      - name: Install dependencies
+        run: |
+          pip install -r requirements.txt
+          pip install PyGithub openai anthropic pyyaml requests
+          
+      - name: Run Hygiene Orchestrator
+        id: hygiene
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:21:40: Double quote to prevent globbing and word splitting

(shellcheck)